instructions/docs/part_1.md

```diff
@@ -0,0 +1,40 @@
+## Part 1: Sort the list of movies (15 points)
+
+On the list of all movies page, make the column headings for "Movie Title" and "Release Date" into clickable links. Clicking one of them should cause the list to be reloaded but sorted in ascending order on that column. For example, clicking the "release date" column heading should redisplay the list of movies with the earliest-released movies first; clicking the "title" header should list the movies alphabetically by title. (For movies whose names begin with non-letters, the sort order should match the behavior of `String#<=>`.)
+
+When the listing page is redisplayed with sorting-on-a-column enabled, the column header that was selected for sorting should appear with a yellow background, as shown below. You should do this by setting controller variables that are used to conditionally set the CSS class of the appropriate table heading to `hilite`, and pasting this simple CSS into RottenPotatoes `app/assets/stylesheets/default.css` file:
+
```

Comment:
this needs to be changed, per email lsat night

Reply:
I plan to merge the Bootstrap change **after** the deadline because removing the 'hilite' class might break the autograder.